src/utils/utils.ts

```diff
@@ -114,22 +117,53 @@ export async function setupNpmProject(folderPath: string): Promise<void> {
     return resolve()
   })
 }
+export async function setupGhooks(folderName: string) {
+  const folderPath = path.join(process.projectDir, folderName)
+
+  process.logger?.info('Installing ghooks')
+  shelljs.exec(`cd ${folderPath} && npm i ghooks --save-dev`, {
+    silent: true
+  })
+
+  const packageJsonPath = path.join(folderPath, 'package.json')
+  const packageJson = require(packageJsonPath)
+
+  if (!packageJson.config) packageJson.config = {}
+
+  if (!packageJson.config.ghooks) packageJson.config.ghooks = {}
+
+  if (packageJson.config.ghooks['pre-commit']) {
```

Comment:
maybe we can also have a check here to prevent it being installed if `sasjs lint` already exists in the pre-commit?

Reply:
fixed